Lazydocs: API Reference doc refresh (#1424)
wandb/docs: content/en/guides/integrations/huggingface.md

The `WandbCallback` that `Trainer` uses will call `wandb.init` under the hood when `Trainer` is initialized. You can alternatively set up your runs manually by calling `wandb.init` before the`Trainer` is initialized. This gives you full control over your W&B run configuration.

> An example of what you might want to pass to `init` is below. For more details on how to use `wandb.init`, [check out the reference documentation]({{< relref "/ref/python/sdk/functions/init" >}}).

Reviewer: ```suggestion
This example demonstrates passing options to `wandb.init`. For more details on how to use `wandb.init`, [check out the reference documentation]({{< relref "/ref/python/sdk/functions/init" >}}).
```
